paper-icon-item.js

```diff
@@ -26,31 +38,27 @@ node with the slot name `item-icon` is placed in the icon area.
 
 The following custom properties and mixins are available for styling:
 
-Custom property               | Description                                    | Default
+Custom property               | Description                                    |
+Default
 ------------------------------|------------------------------------------------|----------
-`--paper-item-icon-width`     | Width of the icon area                         | `56px`
-`--paper-item-icon`           | Mixin applied to the icon area                 | `{}`
-`--paper-icon-item`           | Mixin applied to the item                      | `{}`
-`--paper-item-selected-weight`| The font weight of a selected item             | `bold`
-`--paper-item-selected`       | Mixin applied to selected paper-items                | `{}`
-`--paper-item-disabled-color` | The color for disabled paper-items             | `--disabled-text-color`
-`--paper-item-disabled`       | Mixin applied to disabled paper-items        | `{}`
-`--paper-item-focused`        | Mixin applied to focused paper-items         | `{}`
-`--paper-item-focused-before` | Mixin applied to :before focused paper-items | `{}`
+`--paper-item-icon-width`     | Width of the icon area                         |
+`56px`
+`--paper-item-icon`           | Mixin applied to the icon area                 |
+`{}`
+`--paper-icon-item`           | Mixin applied to the item                      |
+`{}`
+`--paper-item-selected-weight`| The font weight of a selected item             |
+`bold`
+`--paper-item-selected`       | Mixin applied to selected paper-items | `{}`
+`--paper-item-disabled-color` | The color for disabled paper-items             |
+`--disabled-text-color`
+`--paper-item-disabled`       | Mixin applied to disabled paper-items        |
+`{}`
+`--paper-item-focused`        | Mixin applied to focused paper-items         |
+`{}`
+`--paper-item-focused-before` | Mixin applied to :before focused paper-items |
+`{}`
```

Comment:
This table got messed up by the formatter; @e111077 has a fix for this [here](https://github.com/PolymerElements/app-media/blob/90ff9d26ce6436c64d00df3238ece3882d4b5b6c/formatconfig.json), which turns off comment formatting.